moncli/api_v2/handlers.py

```diff
@@ -467,6 +467,68 @@ def create_column(board_id: str, title: str, column_type: ColumnType, *args, **k
     return execute_query(api_key=kwargs.pop('api_key', None), query_name=CREATE_COLUMN, operation_type=gql.OperationType.MUTATION, fields=args, arguments=kwargs)
 
 
+def change_column_title(title: str, column_id: str, board_id: str, *args, **kwargs):
+    """Change an item's column title.
+
+        Parameters
+        
+            title : `str`
+                The new title of the column.
+            column_id : `str`
+                The column's unique identifier.
+            board_id : `str`
+                The board's unique identifier.
+            args : `tuple`
+                The list of item return fields.
+            kwargs : `dict`
+                Optional arguments for changing a column value.
+
+        Returns
+            
+            data : `dict`
+                A monday.com column in item form.
+
+        Return Fields
+        
+            assets : `list[moncli.entities.Asset]`
```

Comment:
Please use the return fields for the Column entity.  These look to be the Item return fields.